otter/models/cass.py

```diff
@@ -206,7 +206,7 @@ def serialize_json_data(data, ver):
 
 _cql_count_for_tenant = (
     'SELECT COUNT(*) FROM {cf} '
-    'WHERE "tenantId" = :tenantId;')
+    'WHERE "tenantId"=:tenantId {deleting};')
```

Comment:
They do not want to know deleting groups. They want count of non-deleting groups. The code is also filtering with `deleting=false`.